lua/telescope/builtin/internal.lua

```diff
@@ -560,15 +560,36 @@ internal.buffers = function(opts)
   }):find()
 end
 
+local colorscheme_previewer = previewers.new({
+	setup = function (self)
+		-- save original colorscheme
```

Comment:
setup needs to return a table, basically a state table. (Setup will always run unless it finds a table)